docs/launch_spec_experiments.py

```diff
@@ -0,0 +1,128 @@
+import os
+import sys
+from uuid import UUID
+
+from gem5art.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art.tasks.tasks import run_gem5_instance
+
+experiments_repo = Artifact.registerArtifact(
+    command = '',
+    typ = 'git repo',
+    name = 'experiment',
+    path =  './',
+    cwd = './',
+    documentation = 'local repo to run spec with gem5'
+)
+
+gem5_repo = Artifact.registerArtifact(
+    command = 'git clone https://gem5.googlesource.com/public/gem5',
+    typ = 'git repo',
+    name = 'gem5',
+    path =  'gem5/',
+    cwd = './',
+    documentation = 'cloned gem5 master branch from googlesource on Nov 12nd'
+)
+
+gem5_binary = Artifact.registerArtifact(
+    command = 'scons build/X86/gem5.opt -j60',
+    typ = 'gem5 binary',
+    name = 'gem5',
+    cwd = 'gem5/',
+    path =  'gem5/build/X86/gem5.opt',
+    inputs = [gem5_repo,],
+    documentation = 'compiled gem5 binary right after downloading the source code'
+)
+
+m5_binary = Artifact.registerArtifact(
+    command = 'make -f Makefile.x86',
+    typ = 'binary',
+    name = 'm5',
+    path =  'gem5/util/m5/m5',
+    cwd = 'gem5/util/m5',
+    inputs = [gem5_repo,],
+    documentation = 'm5 utility'
+)
+
+packer = Artifact.registerArtifact(
+    command = '''wget https://releases.hashicorp.com/packer/1.4.5/packer_1.4.5_linux_amd64.zip;
+    unzip packer_1.4.5_linux_amd64.zip;
+    ''',
+    typ = 'binary',
+    name = 'packer',
+    path =  'disk-image/packer',
+    cwd = 'disk-image',
+    documentation = 'Program to build disk images. Downloaded sometime in November from hashicorp.'
+)
+
+disk_image = Artifact.registerArtifact(
+    command = './packer build spec2006/spec2006.json',
+    typ = 'disk image',
+    name = 'spec2006',
+    cwd = 'disk-image/',
+    path = 'disk-image/spec2006/spec2006-image/spec2006',
+    inputs = [packer, experiments_repo, m5_binary,],
+    documentation = 'Ubuntu Server with SPEC 2006 installed, m5 binary installed and root auto login'
+)
+
+linux_repo = Artifact.registerArtifact(
+    command = '''git clone git clone --branch v4.19.83 --depth 1 https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/''',
+    typ = 'git repo',
+    name = 'linux-4.19.83',
+    path =  'linux-4.19.83',
+    cwd = './',
+    documentation = 'Linux kernel 4.19 source code repo obtained in November'
+)
+
+linux_binary = Artifact.registerArtifact(
+    name = 'vmlinux-4.19.83',
+    typ = 'kernel',
+    path = 'linux-4.19.83/vmlinux-4.19.83',
+    cwd = './',
+    command = '''cp linux-configs/config.4.19.83 linux-4.19.83/.config
+    cd linux-4.19.83
+    make -j8
+    cp vmlinux vmlinux-4.19.83
+    ''',
+    inputs = [experiments_repo, linux_repo,],
+    documentation = "kernel binary for v4.19.83",
+)
+
+
+run_script_repo = Artifact.registerArtifact(
+    command = '',
```

Comment:
There's not enough documentation on this artifact for me to know what it is. Maybe it's OK for the tutorial but should be updated in gem5art-experiments?